src/app/campaign-details/campaign-details.component.html

```diff
@@ -23,6 +23,11 @@ <h3>Campaign Targets</h3>
     <p *ngIf="campaign.amountRaised > 0">Amount Raised: {{ campaign.amountRaised | currency : '£' : 'symbol' : '1.0-0' }}</p>
     <p>Donations: {{ campaign.donationCount }}</p>
 	</div>
+	
```

Comment:
@tbg-jasonfung  this was still using tabs, I guess because the editorconfig was not respected when the file was first made and VS Code follows the file's existing convention. I'll update and merge.